x-pack/plugins/canvas/public/components/app/index.js

```diff
@@ -44,7 +45,8 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
 
 export const App = compose(
   connect(mapStateToProps, mapDispatchToProps, mergeProps),
-  withProps(() => ({
-    onRouteChange: () => undefined,
+  withKibana,
```

Comment:
@crob611 this was your addition but I'm not sure it's needed? We're already being wrapped with a `KibanaContextProvider` right?